dotnet/samples/Demos/ModelContextProtocolPlugin/Program.cs

```diff
@@ -70,3 +71,16 @@
 var prompt = "Summarize the last four commits to the microsoft/semantic-kernel repository?";
 var result = await kernel.InvokePromptAsync(prompt, new(executionSettings)).ConfigureAwait(false);
 Console.WriteLine($"\n\n{prompt}\n{result}");
+
+// Define the agent
+ChatCompletionAgent agent = new()
+{
+    Instructions = "Answer questions about GitHub repositories.",
+    Name = "GitHubAgent",
+    Kernel = kernel,
+    Arguments = new KernelArguments(new PromptExecutionSettings() { FunctionChoiceBehavior = FunctionChoiceBehavior.Auto() }),
+};
+
+// Respond to user input, invoking functions where appropriate.
+ChatMessageContent response = await agent.InvokeAsync("Summarize the last four commits to the microsoft/semantic-kernel repository?").FirstAsync();
+Console.WriteLine($"\n\nResponse from GitHubAgent:\n{response.Content}");
```

Comment:
May be simpler just use the ToString() implicit here.

```suggestion
var response = await agent.InvokeAsync("Summarize the last four commits to the microsoft/semantic-kernel repository?").FirstAsync();
Console.WriteLine($"\n\nResponse from GitHubAgent:\n{response}");
```
```suggestion
ChatMessageContent response = await agent.InvokeAsync("Summarize the last four commits to the microsoft/semantic-kernel repository?").FirstAsync();
Console.WriteLine($"\n\nResponse from GitHubAgent:\n{response.Content}");
```